manifest.xml

```diff
@@ -3,7 +3,7 @@
   This package integrates the gazebo simulator and the Rock framework.
   </description>
   <author>Thomio Watanabe/thomio.watanabe@fieb.org.br</author>
-  <license></license>
+  <license>GNU GPLv2</license>
```

Comment:
Don't mix this, which is pretty important, with a commit that has nothing to do with it !